### DISCUSSION TITLE
Looking forward to covers and embbeded word documents in Quarto Book MS Word

### DISCUSSION THREAD

**Discussion by aito123** [quarto-dev/quarto-cli#1011] [Q&A]
I'm using quarto to write my thesis and it's important to have a specific cover with images, also in the next page I need to insert an abstract with a smaller font than the general text. So my questions are:

1. Is there a way to customize a cover for the Word output of my Quarto Book?
2. Is there a way to set different font sizes along my document?

For example, in the r packages `officedown `& `officer` of David Gohel, it was possible to insert a Word Document in the render of the document, so if i already have a Word Document, i can join the pages in the beginning. Also it's possible to modify the text color and size of a word or sentence. Maybe quarto could bring some functionalities of this amazing package.

Thanks for listening,
Regards

**Comment by jjallaire** [comment]
Thanks for the feedback -- we don't currently have a way to include full docx fragments but this is something we could definitely implement in the future. 

It is technically possible to inject openxml formatting by writing Lua filters w/ Pandoc raw inlines/blocks, but this _very_ advanced so not something we recommend unless you already have experience w/ Lua and openxml. For example, here we inject some openxml at the beginning of a paragraph to set the alignment and padding of an image caption: https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/filters/layout/docx.lua#L27-L29

**Comment by aito123** [comment]
Let me know if it'll be part in a next update. Also i have some ideas that could make better the MS Word export:
I think the goal could be that someone could write an academic journal with the format, the logo, the cover, and all that so, to achieve this we could:
- Insert a cover page with the title, author, abstract, institution logo.
- Change the way numeration appear for headings (e.g: 1. Introduction instead of 1 Introduction)
- A way to call specific MS styles for text, for example when I write `# Heading` it calls the MS Word style Header 1 which I can customize, but what if i want to use another invented style with a specific configuration?

I believe some r packages that have tried to achieve what i have said are: officer, officedown, thesisdown
So maybe Quarto could adapt this solution to fit academic needs in MS Word.

I hope this is a useful feedback,
Regards

**Reply by mcanouil** [reply]
This seems to be related to #170 and basically is an implementation of <https://github.com/rstudio/rticles> into Quarto.

**Comment by jjallaire** [comment]
@aito123 These are great ideas! Thank you. When we return to making improvements in docx in a few months we will definitely take a close look at doing all of these things!
